pkg/features/antrea_features.go

```diff
@@ -163,6 +163,10 @@ const (
 	// alpha: v2.1
 	// Enable the NodeLatencyMonitor feature.
 	NodeLatencyMonitor featuregate.Feature = "NodeLatencyMonitor"
+
+	// alpha: v2.1
+	// Allow users to advertise Service IPs, Pod IPs, and Egress IPs to external BGP peers.
+	BGPPolicy featuregate.Feature = "BGPPolicy"
```

Comment:
Does it support windows?

Reply:
At least IPv4 is supported.

Reply:
Gobgp can be run on Windows.

Reply:
I feel we may simply mention it's not supported yet on Windows if we didn't verify it. Or at least warn user that we didn't do BGP e2e on Windows with IPv4

Reply:
Make sense. I will mention it in documents and comments.

Reply:
I disabled the feature gate in Windows since we don't test it on Windows at all.